src/assets/scripts/client/game/ScoreLogic.js

```diff
@@ -0,0 +1,274 @@
+import EventBus from '../lib/EventBus';
+import GameController, { GAME_EVENTS } from './GameController';
+import UiController from '../ui/UiController';
+import { AIRCRAFT_EVENT } from '../constants/eventNames';
+import { nm_ft } from '../utilities/unitConverters';
+import AirportController from '../airport/AirportController';
+import { speech_say } from '../speech';
+import { AIRPORT_CONTROL_POSITION_NAME } from '../constants/airportConstants';
+import { MCP_MODE } from '../aircraft/ModeControl/modeControlConstants';
+
+/**
+ * @class ScoreLogic
+ */
+export default class ScoreLogic {
+    constructor(aircraftController) {
+        /**
+         * Whether the aircraft has received a clearance to conduct an approach to a runway
+         *
+         * @for ScoreLogic
+         * @property _aircraftController
+         * @type {boolean}
+         * @private
+         */
+        this._aircraftController = aircraftController;
+
+        this.init()
+            .setupHandler()
+            .enable();
+    }
+
+    init() {
+        return this;
+    }
+
+    /**
+     * @for ScoreLogic
+     * @method setupHandler
```

Comment:
`setupHandler` -> `setupHandlers` (should be plural)

Reply:
done